.github/workflows/pr_checks.yml

```diff
@@ -3,26 +3,36 @@ on:
   pull_request:
 
 jobs:
-  backend-unit-tests:
-    name: Backend Unit Tests
+  tests:
+    name: Containerized Tests
     runs-on: self-hosted
     env:
-      BUILD_DIR: tmp/pr_builds/PR_${{ github.event.number }}/backend_unit_tests
+      BUILD_DIR: tmp/pr_builds/PR_${{ github.event.number }}/${{ github.run_id }}
```

Comment:
Not sure if this matters but in case multiple runs need to access the same code we can avoid collisions by having the run id in the path